core-bundle/contao/drivers/DC_Table.php

```diff
@@ -5169,7 +5171,12 @@ protected function sortMenu()
 				$aria_label = \sprintf($GLOBALS['TL_LANG']['MSC']['list_orderBy'], $options_label);
 			}
 
-			$options_sorter[$sortKey] = '  <option value="' . StringUtil::specialchars($value) . '"' . (((!isset($session['sorting'][$this->strTable]) && $field == $firstOrderBy) || $value == $sessionValue) ? ' selected="selected"' : '') . ' aria-label="' . $aria_label . '">' . $options_label . '</option>';
+			$options_sorter[$sortKey] = array(
+				'label' => $options_label,
+				'aria_label' => $aria_label,
+				'value' => StringUtil::specialchars($value),
```

Comment:
No double encoding. 😉
```suggestion
				'value' => $value,
```